app/views/filters/edit.html.erb

```diff
@@ -1,11 +1,23 @@
 <%=
 breadcrumbs(
+  resource_url: api_filters_path(),
+  resource_filter:'role_id=%s' % @filter.role.id ,
+  name_field:"resource_type",
   items: [
-    (@role.present? ? { caption: _('Roles'), url: (url_for(roles_path) if authorized_for(hash_for_roles_path)) } : { caption: _("Filters"), url: filters_path }),
-    ({ caption: _("%s filters") % @role.to_s, url: filters_path(role_id: @role.id) } if @role.present?),
-    { caption: _('Edit %s') % @filter.to_s },
+    {
+      caption: _("Roles"),
+      url: (url_for(roles_path) if authorized_for(hash_for_roles_path))
+    },
+    { 
+      caption: _('%s Filters') % @filter.role.name,
+      url:(filters_path(:role_id => @filter.role) if authorized_for(hash_for_filters_path))
+    },
+    {
+      caption: _('Edit %s Resource Filter' ) % _(@filter.resource_type)
```

Comment:
The drop down doesn't show the translated names - 
![image](https://user-images.githubusercontent.com/1540531/55290347-cedbd480-53da-11e9-8084-cd83142274ca.png)

Reply:
As the resource name comes from the api, I've added a translate in BreadcrumbSwitcherPopover.js